src/java/picard/sam/DownsamplingIterator.java

```diff
@@ -0,0 +1,125 @@
+package picard.sam;
```

Comment:
I guess it could be; I just extracted it from DownsampleSam and put it into the same package.  Want me to shift it over?